src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -517,3 +517,44 @@
     text: |-
         az apim api versionset delete --resource-group MyResourceGroup --service-name MyServiceName --version-set-id MyVersionSetId
 """
+
+helps['apim api create schema'] = """
+type: command
+short-summary: Create an API Management API Schema.
```

Comment:
Add a long-summary that tells user that they must specify either --schema-path or --schema_content

Reply:
added long summary